pymc3/sampling.py

```diff
@@ -134,6 +151,20 @@ def sample(draws, step=None, init='ADVI', n_init=200000, start=None,
         in the system - 2.
     tune : int
         Number of iterations to tune, if applicable (defaults to None)
+    step_kwargs: dict
```

Comment:
It's a bit hacky, I agree. And `nuts_kwargs` actually leads to acceptable API. Let's go with that.